ui/src/pages/frontsite/components/SeriesCard/SeriesCard.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Card, Image, Label } from 'semantic-ui-react';
+import get from 'lodash/get';
+import { getCover } from '@pages/frontsite/config';
+
+export class SeriesCard extends Component {
+  render() {
+    const { data } = this.props;
+    const moi = data.metadata.mode_of_issuance;
```

Comment:
I think we should only display "series", for the end user the "deep" domain language might not be understandable